discord/ext/ipcx/server.py

```diff
@@ -137,10 +142,15 @@ async def handle_accept(self, request: aiohttp.web.Request):
         """
         self.update_endpoints()
 
-        websocket = aiohttp.web.WebSocketResponse()
+        websocket = aiohttp.web.WebSocketResponse(max_msg_size=self.max_msg_size)
```

Comment:
It would be more appropriate instead to raise an error instead of allowing the user to handle this. It is always recommended to abstract this niche portion out instead of allowing the user to shoot themselves in the foot